src/card.css

```diff
@@ -0,0 +1,124 @@
+.page {
+  font-family: "Roboto", sans-serif;
+}
+
+.page__body {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  width: 166px;
+
+  padding-left: 16px;
+  padding-right: 16px;
+  padding-top: 32px;
+  padding-bottom: 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image-wrap {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  padding: 0 3px;
+  padding-bottom: 40px;
+}
+
+.card__image {
+  height: 134px;
+  width: 160px;
+}
+
+.card__title {
+  margin: 0;
+  max-width: 166px;
+  padding-bottom: 4px;
+
+  color: #060b35;
+  line-height: 18px;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.card__code {
+  margin: 0;
+  padding-bottom: 16px;
+
+  line-height: 14px;
+  color: #616070;
+  font-size: 10px;
+  font-weight: 400;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+
+  padding-bottom: 24px;
+}
+
+.card__reviews {
+  margin: 0;
+  display: flex;
+  align-items: flex-end;
+
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+  font-weight: 400;
```

Comment:
You can add these styles to .card and replace some styles when needed. In this case, you will have fewer lines of code and repeats